docs/changes.rst

```diff
@@ -22,6 +22,17 @@ You should generally assume that an API is internal unless you have specific
 information to the contrary.
 
 -------------------
+3.18.1 - 2017-08-15
+-------------------
+
+This release fixes a bug introduced in the previous release. If both
```

Comment:
It's now the previous but one release (I snuck in a 3.18.1 in between). Probably better to mention the version number explicitly  anyway.